debian/changelog

```diff
@@ -1,3 +1,9 @@
+kano-i18n (3.15-1) unstable; urgency=low
```

Comment:
*3.15.0

Reply:
Oh, yeah, we want a third digit. I'll add one.